benchmark/kvbench/commands/nixlbench.py

```
@@ -228,13 +228,31 @@ def _configure_ucx(self, backend: str, source: str, destination: str):
+            "memory": "DRAM",
+            "gpu": "VRAM",
+        }
+
+        src_local = source in arg_to_seg_type
+        dst_local = destination in arg_to_seg_type
+
+        if src_local and dst_local:
+            raise ValueError(
+                f"Invalid source/destination for OBJ: source={source}, destination={destination}, "
+                f"both are local — exactly one must be a remote OBJ endpoint"
+            )
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)